### ISSUE TITLE
Question about how to enable compression and denoise

### ISSUE THREAD

**Issue by birdmanmandbir** [goatchurchprime/two-voip-godot-4#31]
I'm not sure how to turn on compression and denoising based on the Minimal Use Case section of the README. It seems like the only way is to use AudioStreamGeneratorPlayback instead of AudioStreamOpusChunked, also, we should use chunk_to_opus_packet instead of read_opus_packet? This is because chunk_to_opus_packet has a denoise option, but read_opus_packet does not.

Also, it looks like I will need the func resamplerecordedsamples from the radiomqtt example. This will help me get the audio samples I need for the second argument of chunk_to_opus_packet?

I feel that radiomqtt is too complex for a beginner to understand how this plugin works.

**Comment by goatchurchprime** [comment]
Yes, it's pretty confusing, isn't it?  (BTW have you ever made additional docs as asked here?   https://github.com/goatchurchprime/two-voip-godot-4/issues/16   I've not decoded the instructions yet.)
  
Basically, the purpose of this example app is to reproduce and isolate every single nasty audio bug and setting I'm encountering (and there have been a lot of them so far).  Until now it has been too early to make a good minimal use case.  I have only this week got it working on Windows by setting `mix_rate=AudioServer.get_mix_rate()`.

As you have seen, there are two routes through the class: 
   (1) As an integrated audio effect attached to the AudioBus
   (2) As an off-line computation object not attached to an AudioBus that you send PackedVector2Arrays() to.

Route (1) is the runtime optimal case, and it's complicated because we want to minimize the unnecessary computation for the given options.  For example, if the Microphone is going to be gated by a volume threshold, we can measure and act on that sound threshold without applying the resampler, denoiser or opus compression to every frame.  Therefore those later computations need to be deferred until we decide the microphone is on.  (In fact I can go back in time and grab a few frames from before the gate was hit, so as to avoid clipping at the front.)

Route (2) is necessary for the implementation this example app where you can keep changing the opus compression settings and redo the compression on your pre-recorded voice and hear to how it distorts it.

I need to know what's the purpose of a minimal use case to you?  Is it to get familiar with the library and experiment with its features, in which case this example app implements it all.  Or is it as a starting point to embedding it in your app.  I happen to believe that the Vox (Voice activated PTT) feature is essential, but it's pretty complicated, so does it belong in the minimal use case or not?  If you are using this for multiplayer, then https://github.com/goatchurchprime/godot_multiplayer_networking_workbench has the twovoip embedded in it and would start from a higher bar.

**Comment by birdmanmandbir** [comment]
Thank you for your explanation! I'm still a beginner with Godot and audio processing, so I need to switch to Unity to speed things up. I appreciate your hard work, and I hope you can make TwoVoIP and Godot_Multiplayer_Networking_Workbench outstanding projects!
